test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -207,31 +207,33 @@ func clearEnv() {
 func getValidValuesTests(toolToTest string) []latencyTest {
 	var testSet []latencyTest
 
-	//testRuntime: let runtime be 10 seconds for most of the tests and not less, that is to let the tools
-	//have their time to measure latency properly hence stabilizing the tests
+	//testRuntime: for tests with success message (hence anticipated to run the tools),let runtime be 30 seconds for most of the tests for two reasons:
+	//1. to let the tools have their time to measure latency properly 2. have time to check that the pod phase turned running and not completed immediately
 	//testCpus: for tests that expect a success output message, note that an even CPU number is needed, otherwise the test would fail with SMTAlignmentError
-	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "2", outputMsgs: []string{success}, toolToTest: toolToTest})
-	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "6", outputMsgs: []string{success}, toolToTest: toolToTest})
-	testSet = append(testSet, latencyTest{testDelay: "1", testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
-	testSet = append(testSet, latencyTest{testDelay: "60", testRun: "true", testRuntime: "2", testMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
+	successRuntime := "30"
```

Comment:
do we want to use the same value for all the tests? previously it was not the case.

Reply:
yes, this is part of stabling the tests, the value is used for tests that expect a success message hence the tool is executed, it can be different from one test to another but we want to retain the runtime relevant to manage other checks in the main test like verifying the pod gets to running phase. so just for simplicity I used the same value for these tests.